packages/react/src/components/Modal/Modal-story.js

```diff
@@ -79,23 +90,30 @@ storiesOf('Modal', module)
   )
   .add(
     'Trap Focus',
-    () => (
-      <>
-        <Modal {...props()} selectorPrimaryFocus="#text-input-2">
-          <TextInput
-            id="text-input-1"
-            labelText="Text Input 1"
-            placeholder="Enter text..."
-            style={{ marginBottom: '1rem' }}
-          />
-          <TextInput
-            id="text-input-2"
-            labelText="Text Input 2"
-            placeholder="Enter text..."
-          />
-        </Modal>
-      </>
-    ),
+    () => {
+      const { size, ...rest } = props();
+      return (
+        <>
```

Comment:
probably we don't need fragment here anymore right?

Reply:
Good thoughts @emyarod - Seems that before-change code doesn't need the fragment even - And thus I'd like to keep here intact in order to keep the change minimum. (You'll see the difference if click on the gear icon at the top and check off "Hide whitespace changes")